lib/elements/dom-repeat.html

```diff
@@ -580,7 +580,7 @@
       }
       let beforeRow = this.__instances[instIdx + 1];
       let beforeNode = beforeRow ? beforeRow.children[0] : this;
-      this.parentNode.insertBefore(inst.root, beforeNode);
+      inst.insertBefore(this.parentNode, this);
```

Comment:
Should be `inst.insertBefore(this.parentNode, beforeNode);` (which will also solve your eslint error!!)